### DISCUSSION TITLE
Does orlean support net6.0 TargetFramework Now?

### DISCUSSION THREAD

**Discussion by SimTsai** [dotnet/orleans#7488] [Q&A] [answered]
When i build HelloWorld samples, Change TargetFramework to net6.0, build is fail.
if not support now, does roadmap for that?

**Comment by suraciii** [comment, marked as answer]
I suggest you delete or change the `global.json` file and try again, it forces using net5 sdk.

**Comment by dodyg** [comment]
You can find more samples for Orleans here https://github.com/dodyg/practical-aspnetcore/tree/net6.0/projects/orleans using Minimal API on .NET 6
